Changing degree to bachelor's degree across the site (#4566)
DFE-Digital/get-into-teaching-app: app/views/content/train-to-be-a-teacher/if-you-dont-have-a-degree.md

    text: "You can normally only do a teacher degree apprenticeship (TDA) if you've been a resident in the UK for the last 3 years or more. There are some exceptions to this. For example, if you've applied to the Afghan or Ukraine resettlement schemes. Speak to providers for more information."    
---

> ## Bachelor's degree courses

Reviewer: I wonder if this is better left as `Undergraduate` ie not postgraduate? But disagree if needed!

> You need a bachelor’s degree in any subject to train to teach in primary, secondary and special schools in England. If you do not already have a bachelor's degree, you can train to be a teacher as part of your bachelor’s degree and get [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).

Reviewer: ```suggestion
You need a bachelor’s degree in any subject to train to teach in primary, secondary and special schools in England. If you do not already have one, you can train to be a teacher as part of a bachelor’s degree and get [qualified teacher status (QTS)](/train-to-be-a-teacher/what-is-qts).
```
Is it worth taking out one of the `bachelor's` references - it is quite repetitive? But disagree if needed!

$international$

> ## Bursaries for bachelor's degrees

Author: ```suggestion
## Bursaries for undergraduate degrees
```

You may be eligible for a [bursary of $bursaries_undergraduate$](https://www.gov.uk/government/publications/funding-initial-teacher-training-itt/funding-initial-teacher-training-itt-academic-year-2025-to-2026#undergraduate-bursary) if you do a secondary maths or physics course that leads to qualified teacher status (QTS).

> You may also be eligible if you do an opt-in QTS course in secondary computing, languages, mathematics or physics. This is where you start a standard bachelor's degree but then opt-in to initial teacher training to get QTS.

Author: ```suggestion
You may also be eligible if you do an opt-in QTS course in secondary computing, languages, mathematics or physics. This is where you start a standard undergraduate degree but then opt-in to initial teacher training to get QTS.
```
